src/core/core.pri

```diff
@@ -4,12 +4,14 @@ HEADERS += \
     $$PWD/models/downloadsmodel.h \
     $$PWD/models/webdownload.h \
     $$PWD/settings/settings.h \
-    $$PWD/settings/startconfig.h
+    $$PWD/settings/startconfig.h \
+    $$PWD/settings/searchconfig.h
```

Comment:
end with \ so every time you make a change adding or removing a file the diff will only show that line